sql-statements/sql-statement-alter-table-compact.md

```diff
@@ -17,9 +17,11 @@ This data compaction statement is currently supported only for TiFlash replicas,
 
 ```ebnf+diagram
 AlterTableCompactStmt ::=
-    'ALTER' 'TABLE' TableName 'COMPACT' 'TIFLASH' 'REPLICA'
+    'ALTER' 'TABLE' TableName 'COMPACT' ( 'TIFLASH' 'REPLICA' )?
 ```
 
+Since v6.2.0, the `TIFLASH REPLICA` part of the syntax can be omitted. The meaning of the statement remains the same after omission, and takes effect only for TiFlash.
```

Comment:
```suggestion
Since v6.2.0, the `TIFLASH REPLICA` part of the syntax can be omitted. When omitted, the semantic of the statement remains unchanged, i.e. take effect only for TiFlash.
```